continuous-integration/starter-kits/README.md

```diff
@@ -1,6 +1,177 @@
+   [Maven coordinates](https://maven.apache.org/guides/mini/guide-naming-conventions.html). [Configured 
+   URLs to servers](https://maven.apache.org/scm/maven-scm-plugin/usage.html) hosting the 
+   repository are also required. These values rely on the inheritance of a parent configuration. 
+   **Note that values** for the `PARENT_ID` and `PARENT_VERSION` coordinates apply the JPL enterprise 
+   Maven parent to the project. This will vary outside the Lab. 
+   2. Search and replace values in the example [POM file](https://github.com/nasa-ammos/slim/blob/main/continuous-integration/starter-kits/compile-release-publish/java-maven/simple/pom.xml) 
+using the values defined in (i). **See below list of example values.**
+```
+<!-- [INSERT NAME HERE] -->|AMMOS - IDS - SLIM - Sample Projects - Simple
+<!-- [INSERT DESCRIPTION HERE] -->|Sample Projects -- Simple application
+<!-- [INSERT PARENT_GROUP HERE] -->|gov.nasa.jpl.ammos.ids
+<!-- [INSERT PARENT_ID HERE] -->|parent-mipl
+<!-- [INSERT PARENT_VERSION HERE] -->|4.0.1
+<!-- [INSERT GROUP HERE] -->|gov.nasa.ammos.ids.sample_projects
+<!-- [INSERT ID HERE] -->|simple
+<!-- [INSERT SCM_SSH HERE] -->|scm:git:ssh://git@github.com/NASA-AMMOS/slim.git
+<!-- [INSERT SCM_URL HERE] -->|https://github.com/NASA-AMMOS/slim/tree/main
+```
+ 
+5. Modify the Jenkins pipeline script with credentials and relevant configurations. 
+   1. Search and replace values in the example [Jenkinsfile](https://github.com/nasa-ammos/slim/blob/main/continuous-integration/starter-kits/compile-release-publish/java-maven/simple/.ci/Jenkinsfile-buildDeploy) 
+   using the values defined in earlier configuration steps, including credential configuration 
+   in requirements and other determined settings. **See below list of required values, remembering 
+   that these are created during this configuration** 
+```
+/*-- [INSERT ID_OF_STORED_MAVEN_SETTINGS HERE] --*/
+/*-- [INSERT ID_OF_STORED_SSH_KEY HERE] --*/
+/*-- [INSERT URL_TO_GITHUB_SERVER HERE] --*/
+/*-- [INSERT URL_TO_ORG HERE] --*/
+```
+6. Commit changes to the Git repository for testing. 
+7. Create a [Jenkins Multibranch Pipeline Job](https://www.cloudbees.com/blog/jenkins-multibranch-pipeline-with-git-tutorial) 
+configured to point at the Jenkinsfile at `.ci/Jenkinsfile-buildDeploy`
+8. Test and debug using [Jenkins build logging](https://opensource.com/article/19/9/intro-building-cicd-pipelines-jenkins) 
+that reports build progress. **Note: Click the build numbeer to view build logs.**
+ 
+### Maven `multimodule` project
+Note that this differs from the `simple` project by introducing ordered, modular builds, so 
+one repo can build multiple modules. Configuration differences center around the creation of 
+`modules`, or subprojects.
+
+Starter Kit:
+- [Maven Multimodule Project Root](https://github.com/nasa-ammos/slim/blob/main/continuous-integration/starter-kits/compile-release-publish/java-maven/multimodule/)
```

Comment:
Again - recommend we refactor this structure to a [Maven archetype](https://maven.apache.org/guides/mini/guide-creating-archetypes.html) so that its super simple and consistent for people to create the folder structure + files you expect.

Reply:
Also- this link is broken. Can't review as a result.

Reply:
It will be broken until it's merged. The link references another part of the SLIM repo.